packages/mui-base/src/Dialog/Root/DialogRoot.types.ts

```diff
@@ -0,0 +1,112 @@
+export type DialogType = 'dialog' | 'alertdialog';
+
+export type SoftCloseOptions = boolean | 'clickOutside' | 'escapeKey';
```

Comment:
```suggestion
export type SoftCloseOptions = boolean | 'outside-press' | 'escape-key';
```

Let's use kebab-case for reasons?

I also prefer `outside press` to `click outside`, as this is more agnostic for touch/pointer usage, without also using terminology like "pointer outside down" or such which ties it to the event.

Reply:
Prop changed to `dismissible`